src/libstd/yk_swt.rs

```diff
@@ -7,6 +7,45 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
+use ::cell::RefCell;
+use ::fmt;
+
+#[allow(missing_docs)]
+/// A block location in the Rust MIR.
+pub struct MirLoc {
+    pub crate_hash: u64,
+    pub def_idx: u32,
+    pub bb_idx: u32,
+}
+
+impl fmt::Debug for MirLoc {
```

Comment:
Is it worth this manual code vs. `#[derive(Debug)]`?

Reply:
The reason I did a custom one was because the derived one is quite long, with the names of the fields etc.

I wanted a shorter representation.

Reply:
OK, but let's only do this sparingly, just because it's that bit more code to read and maintain.